ramalama/cli.py

```diff
@@ -228,6 +229,23 @@ def configure_arguments(parser):
         action="store_true",
         help="use the model store feature",
     )
+    ip_family_group = parser.add_mutually_exclusive_group()
+    ip_family_group.add_argument(
+        "--ipv4",
+        "-4",
+        dest="ipv4",
+        default=argparse.SUPPRESS,
+        action="store_true",
+        help="use IPv4 connection only"
+    )
+    ip_family_group.add_argument(
+        "--ipv6",
+        "-6",
+        dest="ipv4",
```

Comment:
should not this be `dest=ipv6` ?

Reply:
No, see action below, it will set it to false if that option is present.